### PR TITLE
chore: update nodejs to 18 in gitpod

### PR DESCRIPTION
**Pull request by watany-dev**
I love using Gitpod! I saw that node18 support is needed at https://github.com/aws/aws-cdk/pull/25381 and understood the situation.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by pahud** [comment]
Just opened your PR branch and the prebuild with node18 was doing great.

It's not mandatory to opt in 18 but now that we have a PR here I don't see any reason not 😄.

image

File: .gitpod.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ github:
     pullRequestsFromForks: true
     addComment: true
 
-image: jsii/superchain:1-buster-slim-node16
+image: jsii/superchain:1-buster-slim-node18
```
Comment (pahud) [inline .gitpod.yml:6]: According to https://github.com/aws/jsii/pull/4040 maybe we should use ecr public instead.


```suggestion
image: public.ecr.aws/jsii/superchain:1-buster-slim-node18
```

**Review by pahud — COMMENTED** [review]
Let's use ecr public.

**Comment by watany-dev** [comment]
Thank you for your review. Certainly this should be obtained from ECR Public!

**Review by mrgrain — APPROVED** [review]

**Pull request merged**